README.md

```diff
@@ -16,7 +16,10 @@ the square brackets (for multiple-choice questions).
 Describe the purpose of a clearfix in CSS, and give an example of how to do it.
 
 Your Answer:
-```text
+```
+clearfix is used to clear floats in CSS.
```

Comment:
Would like to see more detail in this answer. See the answer key (which will be shared shortly) for an example answer.